src/ontology/uberon-edit.obo

```diff
@@ -223804,6 +223804,14 @@ intersection_of: located_in UBERON:0001987 ! placenta
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
 property_value: dcterms-date 2022-04-04T11:06:09Z xsd:dateTime
 
+[Term]
+id: UBERON:8470001
+name: sublumbar lymph node
+def: "A lymph node with a lymph center located within the caudal abdomen and pelvis, on the ventral side of the lumbar region. Includes the medial iliac, internal iliac (previously termed hypogastric) and sacral lymph nodes." [https://orcid.org/0000-0002-0819-0473, NCIT:C77655, PMID:32655159]
```

Comment:
@bvarner-ebi
(I know you can't commit this yet, but commenting now as I'll be mostly offline for a couple of days) 
The suggested def says "Includes the medial iliac, internal iliac (previously termed hypogastric) and sacral lymph nodes", so you should look into asserting the following existing terms as subclasses of the new one, please: 
UBERON:0037532 medial common iliac lymph node
UBERON:0015881 internal iliac lymph node
UBERON:0002528 sacral lymph node
Thank you.

Reply:
Thank you for the tip, @paolaroncaglia.
I added 'has part' relationships to these from the new term.

Reply:
@bvarner-ebi from the PMID I can't really tell if sublumbar lymph nodes _always_ "collect" all 3 types listed (the medial iliac nodes are said to be "consistently present", which raises doubts as to whether all 3 types are always present). I did a quick Google search but I'm still unsure. Hence I'd personally be more comfortable with is_a rather than has_part. But I assume that you had evidence to go for has_part. If so, that's fine by me! :-)

Reply:
@paolaroncaglia, thank you for providing more insight on this. I did not have evidence that all 3 types listed are consistently present to form the 'sub lumbar lymph nodes', so I will change these to is_a.